dashboard/app/views/courses/edit.html.haml

```diff
@@ -1,10 +1,10 @@
 - course = local_assigns[:course]
 - course_editor_data = {course_summary: course.summarize,
                         script_names: Script.all.map(&:name),
-                        course_families: Course::FAMILY_NAMES,
-                        version_year_options: Course.get_version_year_options}
+                        course_families: UnitGroup::FAMILY_NAMES,
+                        version_year_options: UnitGroup.get_version_year_options}
 - content_for(:head) do
   %script{ src: webpack_asset_path('js/courses/edit.js'), data: {course_editor: course_editor_data.to_json}}
-= form_for(course) do |f|
+= form_for(course, {url: course_path(course)}) do |f|
```

Comment:
if not specified, `form_for` looks for url path helpers based on the type of the object you pass it, in this case looking for the non-existent `unit_group_path` helper.

Reply:
So when you rename the table will this no longer be needed?

Reply:
not quite - this discrepancy is because the `resources :courses` declaration in routes.rb does not match the `UnitGroup` model name. rails docs for resources are here: https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default . They show that `resources` defines 3 sets of things: url paths, controller actions, and url path helpers. all 3 are customizable, via `path:`, `controller:` and `as:`, respectively. so, one way to fix this differently would be to change the resource declaration to look like this:
```
resources :courses, as: 'unit_group'
```
that would mean the following:
* we have to rename all course url helpers (e.g. course_path --> unit_group_path, courses_path --> unit_group_path)
* we can stop specifying urls for form_for(unit_group) and redirect_to(unit_group), because they'll now infer the right path helper.